plugins/flytekit-vscode/README.md

```diff
@@ -0,0 +1,41 @@
+# Flytekit VSCode Plugin
+
+The Flytekit VSCode plugin offers an easy solution for users to run Python tasks within an interactive VSCode server, compatible with any image. `@vscode` is a decorator which users can put within @task and user function. With `@vscode`, the task will install vscode dependencies (skip if they already exist) and run a vscode server instead of the user defined functions.
+
+To install the plugin, run the following command:
+
+```bash
+pip install flytekitplugins-vscode
+```
+
+## Task Example
+```python
+from flytekit import task
+from flytekitplugins.vscode import vscode
+
+@task
+@vscode
```

Comment:
did you experiment at all with `@task + @vscode`?

Reply:
Do you mean the end to end test of `@task + @vscode`. If yes, then I have done it. You can check the **End to end test** section of the PR description.

Reply:
What does that mean? Could you give an example